src/Alert/index.jsx

```diff
@@ -104,7 +136,8 @@ Alert.defaultProps = {
   stacked: false,
 };
 
-Alert.Link = BaseAlert.Link;
-Alert.Heading = BaseAlert.Heading;
+Alert.Heading = AlertHeading;
+Alert.Link = AlertLink;
 
+export { AlertHeading, AlertLink };
```

Comment:
Are these named exports necessary when they are accessible via `Alert.Link` and `Alert.Heading`?

Reply:
They are actually not, removed them now. For some reason I thought documentation site would not pick up and display props definitions if you don't explicitly export the components, but now after taking a closer look at the code I see that's not how it works :). Thank for pointing it out!